frontend/src/lib/components/WarningTag/WarningTag.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+import { LemonTag } from '../LemonTag/LemonTag'
+
+export function WarningTag({ children = null }: { children: React.ReactNode }): JSX.Element {
```

Comment:
Hmm, this seems pretty redundant considering that `LemonTag` simply has `type="warning"`. Can't we use that without an extra wrapper?